webrtc.html

```diff
@@ -2354,11 +2356,124 @@
 
           <dd>This indicates the index (starting at zero) of the m-line in the
           SDP this candidate is associated with.</dd>
+
+          <dt>DOMString foundation</dt>
+
+          <dd>A unique identifier that allows ICE to correlate candidates that
+          appear on multiple <code><a>RTCIceTransport</a></code>s.</dd>
+
+          <dt>unsigned long priority</dt>
+
+          <dd>The assigned priority of the candidate. This is automatically
+          populated by the browser.</dd>
```

Comment:
The only place where this has any effect on behavior is in addIceCandidate, and that says it ignores these values.